axelor-tool/src/main/resources/i18n/messages_fr.csv

```diff
@@ -1,5 +1,6 @@
 "key","message","comment","context"
 "An error occurs while opening the connection. Please verify the following URL : %s.","Une erreur est survenue pendant la connexion. Veuillez verifier l'URL suivant: %s.",,
+"API calls are disabled, please modify application.properties aos.api.enable field.","Les appels API sont désactivés, veuillez modifier le champ aos.api.enable dans application.properties.",,

```

Comment:
"AOS API calls are disabled, please contact your administrator. The property aos.api.enable is false in application.properties.","Les appels API AOS sont désactivés, veuillez contacter votre administrateur. La propriété aos.api.enable est à faux dans le application.properties."